.github/CONTRIBUTING.md

```diff
@@ -21,3 +21,59 @@
     * shared - for changes at the shared module
     * config - for changes at configuration files
     * \* - for other changes
+
+## How To - Develop
+
+### Setup dev environment
+
+You will need the following tools installed.:
+* [GIT](https://git-scm.com/)
+* [Node and Npm](https://nodejs.org/en/)
+
+We recommend you to use [Visual Studio Code](https://code.visualstudio.com/) as IDE.
+
+#### Clone repository and install dependency's
+
+run
+```
+$ git clone https://github.com/redmedical/hvstr.git
+```
+to clone hvstr on your machine.
+
+Now you need to install all npm packages. Hvstr consists of 3 components:
+* core
+* client
+* utils
+You will need to install the dependency's for each. Run:
+```
+$ cd utils
+$ npm install
+$ cd ../client
+$ npm install
+$ cd ../core
+```
+
+#### Use local version of hvstr-utils
+If you make any changes at the hvstr-utils component and want to use it in the core or client you can install it locally.
+
+First you need to build the utils package.
+```
+$ cd utils
+$ npm run build
+```
+afterwards you can switch in the core or client directory and install the build locally.:
+```
+$ cd ../core
+$ npm install ../utils --no-save
+```
+> make sure, the local installation is not taken over by the `package.json`.
+
+#### Build the project and run tests
+
+To build all components, you will need to run ```npm run build``` in each component. Each component contains the npm scripts to ```build``` and ```lint```. Currently just the core contains any tests. You can run them with ```npm run test```.
+
+#### Debug the project.
+The easiest way is, to install hvstr in a test project and use the debugging tools of the project. If you want to debug the unit tests you can use this manual.:  
```

Comment:
> The easiest way is, to install hvstr in a test project and use the debugging tools of the project.
The easiest way to debug hvst, is to install it in a test project and use the debugging tools of the test project.